packages/@okta/vuepress-site/docs/guides/social-login/main/amazon/ea-icon.md

```diff
@@ -1 +0,0 @@
-<ApiLifecycle access="ea" />
```

Comment:
I'm confused: do these files still need to exist?

Reply:
These snippets are not used now. I thought I'd leave them for flexibility - if we need another flag underneath the title in the future? Let me consult with @timhowie-okta first. I'll remove them if there are no other plans for them (incidentally, I have another ticket to clean up un-used snippets in the Idps: OKTA-459709)

Reply:
Hi @colekurashige-okta @vanngo-okta - originally I'd put that into the template so there's a placeholder there if there's a need to mark any IdP as being EA in the future. IIRC, I think that @johnbaldo-okta had stated that after support for 'additional social IdPs' went GA, that social IdPs added in the future would basically skip the EA step (which I'm guessing implies they'd hit Preview one month and then Prod the following month).

If that is indeed correct, then I don't see an issue with removing the tag from the template - but I can see how it could be used for some other special purpose if some different form of tagging was ever needed. However, I imagine that the template could just as easily be modified to insert a different element at the point, should it ever occur.

Reply:
Yeah, I think it would be good to remove them for now. Could bring it back in the future if we need an EA state, but no plans for that currently.